### PR TITLE
Added link to [Joe Dean](https://joedean.dev)

### PR DESCRIPTION
**Pull request by joedeandev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 A list of developer portfolios for your inspiration
 
-Have you built a portfolio? Are you proud of it?! Open a [PR](./CONTRIBUTING.md) to this repo and let's showcase your work! Refer to the [CONTRIBUTING](./CONTRIBUTING.md) file for direction.
+Have you built a portfolio? Are you proud of it?! Open a [PR](./CONTRIBUTING.md)
+to this repo and let's showcase your work! Refer to the
+[CONTRIBUTING](./CONTRIBUTING.md) file for direction.
```
Comment (eclectic-coding) [inline README.md:7]: Remove these changes

File: README.md [hunk 2]
```diff
@@ -11,7 +13,10 @@ Hopefully this repo can serve as a source of inspiration for your portfolio!
 
 ---
 
-**Jump to:** [A](#a) | [B](#b) | [C](#c) | [D](#d) | [E](#e) | [F](#f) | [G](#g) | [H](#h) | [I](#i) | [J](#j) | [K](#k) | [L](#l) | [M](#m) | [N](#n) | [O](#o) | [P](#p) | [Q](#q) | [R](#r) | [S](#s) | [T](#t) | [U](#u) | [V](#v) | [W](#w) | [Y](#y) | [Z](#z)
+**Jump to:** [A](#a) | [B](#b) | [C](#c) | [D](#d) | [E](#e) | [F](#f) | [G](#g)
+| [H](#h) | [I](#i) | [J](#j) | [K](#k) | [L](#l) | [M](#m) | [N](#n) | [O](#o)
+| [P](#p) | [Q](#q) | [R](#r) | [S](#s) | [T](#t) | [U](#u) | [V](#v) | [W](#w)
+| [Y](#y) | [Z](#z)
```
Comment (eclectic-coding) [inline README.md:19]: Remove these changes

File: README.md [hunk 3]
```diff
@@ -296,7 +302,8 @@ Hopefully this repo can serve as a source of inspiration for your portfolio!
 - [Kamran Hamid](https://mkamranhamid.netlify.com)
 - [Karen Fletcher](https://knpfletcher.dev)
 - [Karthik Menon](https://www.karthikmenon.com/)
-- [Katie Amberg-Johnson](https://kambergjohnson.com) (made with [TechFolios](https://techfolios.github.io))
+- [Katie Amberg-Johnson](https://kambergjohnson.com) (made with
+  [TechFolios](https://techfolios.github.io))
```
Comment (eclectic-coding) [inline README.md:306]: Remove these changes

File: README.md [hunk 4]
```diff
@@ -416,7 +423,8 @@ Hopefully this repo can serve as a source of inspiration for your portfolio!
 - [Pan Wei Lian](https://williamson922.github.io/)
 - [Pawaret Meungkaew](https://www.pawaret.dev/)
 - [Pazindu Shane](https://pazindushane.github.io/)
-- [Philip Johnson](https://philipmjohnson.org) (made with [TechFolios](https://techfolios.github.io))
+- [Philip Johnson](https://philipmjohnson.org) (made with
+  [TechFolios](https://techfolios.github.io))
```
Comment (eclectic-coding) [inline README.md:427]: Remove these changes

File: README.md [hunk 5]
```diff
@@ -468,7 +476,8 @@ Hopefully this repo can serve as a source of inspiration for your portfolio!
 - [Sarang N](https://srng.dev)
 - [Sarvesh Patil](https://sarveshpatil.com)
 - [Sai Teja](https://saiteja13427.github.io/)
-- [Satish Jhanwer](https://satishjhanwer.github.io/) ([@satishjhanwer](https://github.com/satishjhanwer))
+- [Satish Jhanwer](https://satishjhanwer.github.io/)
+  ([@satishjhanwer](https://github.com/satishjhanwer))
```
Comment (eclectic-coding) [inline README.md:480]: Remove these changes

**Review by eclectic-coding — COMMENTED** [review]
Seems like your editor updated a few additional areas. Please revert the areas noted and I will gladly add to site.
Thanks

**Comment by joedeandev** [comment]
I should have checked that [Prettier](https://prettier.io/) wasn't making surprise changes - sorry about that! It should be fixed now.

**Comment by eclectic-coding** [comment]
No problem. Thanks for your interest.

**Pull request merged**